src/index.js

```diff
@@ -0,0 +1,264 @@
+            document.getElementById("tempValue").style.color = "teal";
+        } else if (tempValue <= 0) {
+            document.getElementById("tempValue").style.color = "blue";
+        }
+    }
+};
+
+const findLatitudeAndLongitude = (query) => {
+    let latitude, longitude;
+    axios.get("http://127.0.0.1:5000/location",
+        {
+            params: {
+                q: query,
+                format: "json"
+            }
+        })
+        .then((response) => {
+            latitude = response.data[0].lat;
+            longitude = response.data[0].lon;
+            findWeather(latitude, longitude);
+        })
+        .catch((error) => {
+            locationError = true;
+            const errorSpan = document.createElement("span");
+            errorSpan.id = "invalid"
+            errorSpan.innerHTML = "Invalid City"
+            cityNameSection.appendChild(errorSpan)
+            console.log(error)
+            console.log("error in findLatitudeAndLongitude!");
+        });
+}
+
+const findWeather = (latitude, longitude) => {
+    axios.get("http://127.0.0.1:5000/weather",
+        {
+            params: {
+                format: "json",
+                lat: latitude,
+                lon: longitude
+            }
+        })
+        .then((response) => {
+            const tempK = response.data.main.temp;
+            let temp;
+            if (tempType === "fahrenheit") {
+                temp = Math.floor((tempK - 273.15) * 9 / 5 + 32);
+            } else {
+                temp = Math.floor((tempK - 273.15));
+            }
+
+            tempValue.textContent = temp;
+            updateLandscape(temp);
+            updateTextColor(temp);
+            count = temp;
+
+            const skyData = response.data.weather[0].main;
+            UpdateSkySelect(skyData);
+        })
+        .catch((error) => {
+            console.log(error)
+            console.log("error in findLocation!");
+        });
+}
+
+updateLandscape(count);
+updateTextColor(count);
+
+const changeCity = () => {
+    const cityName = document.getElementById("cityNameInput").value;
+    const cityHeader = document.getElementById("headerCityName");
+
+    cityHeader.textContent = cityName[0].toUpperCase() + cityName.substring(1);
+
+    if (locationError) {
+        const invalid = document.getElementById("invalid")
+        invalid.remove();
+        locationError = false;
+    }
+};
+
+const UpdateSkySelect = (skyData) => {
+    switch (skyData) {
+        case "Clouds":
+            document.getElementById("skySelect").value = "Cloudy";
+            break;
+        case "Rain":
+            document.getElementById("skySelect").value = "Rainy";
+            break;
+        case "Snow":
+            document.getElementById("skySelect").value = "Snowy";
+            break;
+        default:
+            document.getElementById("skySelect").value = "Sunny";
+            break;
+    };
+    updateSky();
+}
```

Comment:
```suggestion
};
```
Assignments need a semicolon at the end, even if the value is a function